.travis.yml

```diff
@@ -7,11 +7,13 @@ node_js:
 
 script:
   - node ./internals/scripts/generate-templates-for-linting
-
-install:
   - npm run test
   - npm run build
 
+install:
+  - npm i -g npm@latest
+  - npm install
+
```

Comment:
If we are committed to installing dependencies with yarn, what purpose does this serve?